downstream/modules/hub/proc-approve-collection.adoc

```diff
@@ -14,7 +14,7 @@ Collections requiring approval have the status *Needs review*. Click the *Versio
 
 .Procedure
 
-. From the sidebar, navigate to menu:Approval[].
+. From the navigation panel, navigate to menu:Approval[].
```

Comment:
Suggest changing to agreed upon phrasing:
From the navigation panel, select menu:Approval[]